src/azure-cli-core/azure/cli/core/aaz/_selector.py

```diff
@@ -55,6 +54,4 @@ def set(self, value):
 
     def required(self):
         value = self.get()
-        if not has_value(value):
-            raise UserFault("ResourceNotFoundError")
         return value
```

Comment:
after change what's the difference between `required` and `get`? I think the correct way is fix the generated code which use required() to get()